test/valgrind/valgrindTestSuite.sh

```diff
@@ -156,7 +156,7 @@ dryrun=off
 leakTest=off
 dryLeaks=off
 fromIx=0
-ixLists=""
+ixList=""
```

Comment:
My guess is that the variable ixList was empty by default ...
Perhaps all shell variables are, and these initializations are not necessary ... ?